qemu/tests/hotplug_mem_repeat.py

```diff
@@ -0,0 +1,78 @@
+import time
+import logging
+from avocado.core import exceptions
+from virttest.utils_test.qemu import MemoryHotplugTest
+
+try:
+    from virttest import error_context as step_engine
+except ImportError:
+    from autotest.client.shared.error import step_engine
+
+
+class MemoryHotplugRepeat(MemoryHotplugTest):
+
+    def start_test(self):
+        """
+        Prepare reqired test params, then start memory
+        hotplug/unplug tests in turn. And this is test entry.
+        """
+        target_mem = self.params["target_mem"]
+        vm = self.env.get_vm(self.params["main_vm"])
+        max_slots = int(self.params.get("slots_mem", 4))
+        scalability_test = self.params.get("scalability_test") == "yes"
+        repeats = scalability_test and max_slots or self.params["repeats"]
+        for repeat in xrange(int(repeats)):
+            extra_params = (scalability_test and
+                            [{'slot_dimm': repeat}] or [None])[0]
+            step_engine.context(
+                "Hotplug/unplug loop '%d'" %
+                repeat, logging.info)
+            self.turn(vm, target_mem, extra_params)
+        vm.verify_alive()
+        vm.reboot()
+
+    def turn(self, vm, target_mem, extra_params=None):
+        """
+        Hotplug/Unplug memory in turn
+
+        :param vm: qemu target VM object
+        :param target_mem: memory name of target VM object
+        :param extra_params: params dict, that you want to update
+
+        """
+        memorys = self.get_all_memorys(vm)
+        if extra_params:
+            self.params.update(extra_params)
+        self.hotplug_memory(vm, target_mem)
+        memorys_added = self.get_all_memorys(vm) - memorys
```

Comment:
really a small bug in PR   https://github.com/avocado-framework/avocado-vt/pull/443. Hi @CongLi and @vivianQizhu would you like pull latest code and retest. 

Thanks,
Xu